RecoBTag/TensorFlow/python/pfDeepDoubleCvBJetTags_cfi.py

```diff
@@ -1,21 +1,12 @@
 import FWCore.ParameterSet.Config as cms
+from pfDeepDoubleBvLJetTags_cfi import pfDeepDoubleBvLJetTags
 
-pfDeepDoubleCvBJetTags = cms.EDProducer('DeepDoubleXTFJetTagsProducer',
-  src = cms.InputTag('pfDeepDoubleXTagInfos'),
-  input_names = cms.vstring(
-    'input_1',
-    'input_2',
-    'input_3'
-  ),
-  graph_path = cms.FileInPath('RecoBTag/Combined/data/DeepDoubleX/94X/V01/DDCvB.pb'),
-  lp_names = cms.vstring('db_input_batchnorm/keras_learning_phase'),
-  output_names = cms.vstring('ID_pred/Softmax'),
-  flavor = cms.string('CvB'),
-  flav_table = cms.PSet(
-    probHbb = cms.vuint32(0),
-    probHcc = cms.vuint32(1)
-  ),
-  batch_eval = cms.bool(False),
-  nThreads = cms.uint32(1),
-  singleThreadPool = cms.string('no_threads')
+pfDeepDoubleCvBJetTags = pfDeepDoubleBvLJetTags.clone(
+    flavor = 'CvB',
+    graph_path = cms.FileInPath('RecoBTag/Combined/data/DeepDoubleX/94X/V01/DDCvB.pb'),
+    flav_table = cms.PSet(
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

For nested parameters it should be ```flav_table = dict ( probHbb = [0], probHcc = [1])``` (not fully sure about the vector initializer).

However, given the fillDescription already contains graph_path and flav_table definition for flavor = CvB, I think that neither graph_path no flav_table modifications are needed

Reply:
I am not sure I understand your point about the dictionary.  

Somehow only specifying flavor and not changing the flav_table to match, results in an error. I also though this would be taken care of with the switch, but removing line 6-10 in this files give:
`   [1] Validating configuration of module: class=DeepDoubleXTFJetTagsProducer label='pfDeepDoubleCvLJetTags'
Exception Message:
Illegal parameter found in configuration.  The parameter is named:
 'probHbb'`